package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pre-commit:ui-react": "cd app/ui-react && yarn pre-commit",
-    "pre-push:ui-react": "syndesis -m ui-react --incremental"
+    "pre-push:ui-react": "./tools/bin/syndesis -m ui-react --incremental"
```

Comment:
what's the assumed working folder? project root? is that always the case?
Isn't there a way to express a location based on the position of `package.json` itself?

Reply:
I see also the other scripts make this assumption, so I assume it's fine